tools/ratbagd.py

```diff
@@ -440,6 +440,26 @@ class RatbagdButton(_RatbagdDBus):
     ACTION_TYPE_KEY = 3
     ACTION_TYPE_MACRO = 4
 
+    ACTION_SPECIAL_UNKNOWN = (1 << 30),
```

Comment:
I'll amend this locally and push, but you forgot the comma at the end of the line, which is a bug you did not see because driver-test.c was not picking up specials from the config input.